kedro-datasets/kedro_datasets/__init__.py

```diff
@@ -1,3 +1,3 @@
 """``kedro_datasets`` is where you can find all of Kedro's data connectors."""
 
-__version__ = "1.0.0"
+__version__ = "1.1.0"
```

Comment:
We only bump the version for a release, not with every new feature added, so this should be reverted.

Reply:
Right, fixing!